draft-ietf-rats-ar4si.md

```diff
@@ -417,19 +417,49 @@ executables:
    : Verifier malfunction
 
    2:
-   : Only a recognized genuine set of approved executables, scripts, files, and/or objects have been loaded during and after the boot process.
+   : Only a recognized genuine set of approved components have been loaded during the boot process.
 
-   3:
-   : Only a recognized genuine set of approved executables have been loaded during the boot process.
+   32:
+   : Only a recognized genuine set of components have been loaded.  However the Verifier cannot vouch for a subset of these due to known bugs or other known vulnerabilities.
+
+   33:
+   : Components loaded at boot-time include executables, scripts, files, and/or objects that are not recognized.
+
+   34:
+   : Components loaded at boot-time include executables, scripts, files, and/or objects that are not recognized, and the Verifier cannot vouch for a subset of the recognized components due to known bugs or other known vulnerabilities.
```

Comment:
Would this not fall under "contraindicated"?  If some components are unrecognized, and those that are recognized contain vulns (i.e. are contrainicated), then take the more severe condition.